harvester_e2e_tests/apis/test_networks.py

```diff
@@ -58,16 +58,30 @@ def test_create_without_name(self, api_client):
         assert 422 == code, (code, data)
         assert "Invalid" == data.get("reason"), (code, data)
 
+    @pytest.mark.parametrize("vlan_id", [4095])
+    @pytest.mark.skip_version_before("v1.1.0")  # ref to harvester/issues/3151
+    def test_create_with_invalid_id_110(self, api_client, unique_name, vlan_id):
```

Comment:
Let's keep the name as `test_create_with_invalid_id` which will run after v1.1.0 and `test_create_with_invalid_id_103` for the test which run with version 1.0.3.